lib/common/hash.go

```diff
@@ -101,3 +101,21 @@ func HexToHash(in string) (Hash, error) {
 	copy(buf[:], out)
 	return buf, err
 }
+
+// MustHexToHash turns a 0x prefixed hex string into type Hash
+// it panics if it cannot turn the string into a Hash
+func MustHexToHash(in string) Hash {
+	if strings.Compare(in[:2], "0x") != 0 {
```

Comment:
Same question, do we need `HexToHash` which returns error, or this which may panic?

Reply:
same as above, it's nice to have a version that doesn't panic